default/web_tt2/skinsedit.tt2

```diff
@@ -6,197 +6,436 @@
 <p><strong>[%|loc%]static css installation succeed. Reload the current page and/or check sympa logs to be sure that static css a really in use.[%END%]</strong></p>
 [% END %]
 
-<h2>[%|loc%]Colors[%END%]</h2>
+<form action="[% path_cgi %]" method="POST">
+<fieldset>
 
-<p>
-[%|loc%]Use the color editor in order to change defined colors. First select the color you want to change and pick a color,then apply it using the test button. The new color is not really installed but it is used only for your own session. When happy with the different colors you chosen, you may save them in a new static CSS.[%END%]
-</p>
+<h2>
+  [%|loc%]Colors[%END%]
+  <a href="#" class="accordionButton" data-selector="#help\.Colors">
+    <i class="fa fa-question-circle" title="[%|loc%]Help[%END%]"></i>
+  </a>
+</h2>
 
-<p>
-[%|loc%]Be careful: the CSS file is overwritten using css.tt2 template, usually this file is coming from Sympa distribution so your CSS customization may be erased when doing this.[% END %]
-[%|loc%]So if you want to preserve some site customizations from being overwriten when upgrading to a new sympa version, css_path and css_url should not point to the same directory.[%END%]
-</p>
+<div id="help.Colors" class="secondary callout" data-closable>
+  <p>
+    [%|loc%]Use the color editor in order to change defined colors. First select the color you want to change and pick a color,then apply it using the test button. The new color is not really installed but it is used only for your own session. When happy with the different colors you chosen, you may save them in a new static CSS.[%END%]
```

Comment:
Missing space, s/color,then/color, then/
Typo, s/you chosen/you have chosen/

Reply:
As I took care not to change existing phrases, it would be corrected in translation catalog (see also description in #115).

Reply:
Indeed the pointer was unclear.
https://github.com/sympa-community/sympa/issues/115#issuecomment-364745915
>   - On [English page of translation site](https://translate.sympa.org/en_US/), add correct string as a "translation" of incorrect msgid.
> 
> Your change on the site above will be reflected to source distribution in the future release.

Reply:
Done.